NickvisionTagger.GNOME/Views/MainWindow.cs

```diff
@@ -672,7 +636,7 @@ private void SwitchAlbumArt(Gio.SimpleAction sender, EventArgs e)
     /// Occurs when the insert album art action is triggered
     /// </summary>
     /// <param name="type">AlbumArtType</param>
-    private void InsertAlbumArt(AlbumArtType type)
+    private async Task InsertAlbumArt(AlbumArtType type)
```

Comment:
`InsertAlbumArtAsync`
*Whatever actions calls this method make sure you add `await` in front*